components/list/list-item-content.js

```diff
@@ -26,13 +26,21 @@ class ListItemContent extends LitElement {
 				overflow: hidden;
 			}
 
+			.d2l-list-item-content-text-supportingInformation {
+				color: var(--d2l-list-item-content-text-color);
+				margin: 0;
+				margin-top: 0.15rem;
+				overflow: hidden;
+			}
+
 		`];
 	}
 
 	render() {
 		return html`
 			<div class="d2l-list-item-content-text d2l-body-compact"><slot></slot></div>
-			<div class="d2l-list-item-content-text-secondary d2l-body-small"><slot name="secondary"></slot></div>
+			<div class="d2l-list-item-content-text-secondary d2l-body-small"><slot name="supportingInformation"></slot></div>
```

Comment:
I think this slot should be named `supporting-information`  for consistency with other multiple-word slots

Reply:
what is a trick for structuring the html this way: div with class secondar has slot 'supporting', and div with class supporting has slot secondary ?

Reply:
> what is a trick for structuring the html this way: div with class secondary has slot supporting, and div with class supporting has slot secondary ?

actualy should be reading PR description. it has an explanation

Reply:
`supporting-info` would be fine too.